doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -15,6 +15,57 @@
+    - It is not necessarily the case that the vendor that the operator chooses for Kubernetes is the same vendor that provides the NFVI and/or VIM for that operator. That is a procurement decision for each operator.
+
+The main pros and cons to these approaches can be seen in the table below. This is broadly similar to the discussion had in the early days of virtualisation about whether or not to share NFVI platforms.
+
+| Option | Pros | Cons | Notes |
+| --- | --- | --- | --- |
+| 1. Software vendor brings Kubernetes | Potentially quicker validation/certification | Potential for silo deployments, operational complexity | ... |
+| 2. Operator brings Kubernetes | Maximum efficiency of infrastructure | Good for those operators building capabilities | ... |
+
+Suggested recommendation:
+- Option 2, operator brings Kubernetes to meet model for other "as-a-Service" components that are required such as DNS, NTP, etc.
+
+Figure 1-1 below shows this at a high level.
+
+<p align="center"><img src="../figures/ch01-k8s-arch.PNG" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="65%"/></p>
+<p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
+
+Other thoughts:
+- Kubernetes / CaaS should be recognised in ETSI NFV / MANO
+- It is feasible that Kubernetes, using operators and custom resources, could become a generic VNFM (i.e. the Kubernetes parts are the underlying engine, with the operators and custom resources being the equivalent of the specific VNFM).
+- Should each operator decide how Kubernetes is shared (if at all)?  Should we/someone suggest some best practice (such as, link Kubernetes cluster lifecycle to application lifecycle - so VNFv1.0 is in one cluster, VNFv2.0 is in another cluster. Are clusters shared between VNF vendors, or VNF types?). 
+
+A note on virtualised and containerised workloads:
+It is highly likely that a single VNFM will be managing applications (or components) that are virtualised (i.e. running in VMs) and applications (or components) that are containerised (i.e. running in containers), at the same time.  There are different approaches to achieving this:
+1. Application manager (EMS, VNFM) uses both IaaS (VIM) API and Kubernetes API - the former for VM based workloads, the latter for containerised workloads.
+2. Application manager (EMS, VNFM) uses just the Kubernetes API for both VM-based and container-based workloads, with the Kubernetes PaaS managing the lifecycle of VMs using on of the following methods:
+    a) Kubernetes interacts with IaaS/VIM API (Cluster API model)
+    b) Kubernetes is the IaaS/VIM and interacts direct with the hypervisor (Kubevirt model)
+
```

Comment:
we need to add the deployment mode,  K8S over IaaS ( Openstack ) or K8s over BareMetal

Reply:
See the later section about bare metal considerations. The general model is the same, even with bare metal we need an IaaS API it's just for a Physical Infra Manager instead of VIM.

Reply:
Good  , but when we mention IaaS doesn't mean BareMetal , so I preferred to mention is a BareMetal that exposed open APIs 
what did u think @tomkivlin

Reply:
More than happy to call it out, will do so when at my desk next. Not sure I agree that IaaS != BareMetal though, agree we can make this clearer here though.